### PR TITLE
Encryption/Decryption RPC interaction flows & tokens

### PR DESCRIPTION
**Pull request by chunningham**
Builds upon #19 , adds encryption/decryption RPC interaction flows

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -172,6 +174,52 @@ export class JolocomSDK {
...
+      {
+        message: {
+          callbackURL: req.callbackURL,
+          rpc: CallType.AsymEncrypt,
+          request: {
+            data: req.toEncrypt.toString('base64'),
+            target: req.target,
+          },
+        },
+        typ: CallType.AsymEncrypt,
```
Comment (Exulansis) [inline index.ts:213]: Do we need to repeat ourselves both in the `rpc` and `typ` values?

Comment (Exulansis) [reply index.ts:213]: Also is the `typ` supposed to be the token type (e.g. `generic(?)`)?

Comment (chunningham) [reply index.ts:213]: ah, kind of leftover cruft from the change from explicitly generic tokens to just tokens. can adjust

**Comment by mnzaki** [comment]
This branch has gone into `feat/un-encdec` and for organisational reasons will close this PR

**Pull request closed without merging**